cmd/k8s-operator/deploy/chart/templates/deployment.yaml

```diff
@@ -34,17 +34,11 @@ spec:
       securityContext:
         {{- toYaml . | nindent 8 }}
       {{- end }}
-      {{- if or .Values.oauth.clientSecret .Values.oauth.audience }}
       volumes:
-        {{- if .Values.oauth.clientSecret }}
+        {{- if .Values.oauthSecretVolume }}
```

Comment:
`.values.oauthSecretVolume` takes precedent above all. If it's set, we want to use it.